MacrobenchmarkSample/README.md

```diff
@@ -38,6 +38,31 @@ Alternatively, run the benchmarks from terminal with:
 ./gradlew macrobenchmark:cC
 ```
 
+### Macrobenchmark with Composition Tracing
+Composition Tracing allows to run system tracing with information on when all Composables (re)compose.
+This gives you insights on where the UI spends majority of the time and helps you find jank.
+
+To set up composition tracing for your app, follow our [documentation](https://developer.android.com/jetpack/compose/tooling/tracing).
+To get composition tracing when running a macrobenchmark, you also need to use `androidx.benchmark.perfettoSdkTracing.enable=true` instrumentation argument.
+
+You can check the `Scroll List With Composition Tracing` run configuration that is part of the project, 
+which runs the scroll compose list benchmark while also recording the information on composition.
+
+It produces results like in the following table:
+```
+FrameTimingBenchmark_scrollComposeList
+%EntryRow (%Count                                    min   5.0,   median   6.0,   max   6.0
+%EntryRow (%Ms                                       min  10.2,   median  11.8,   max  16.2
+EntryRowCustomTraceCount                             min   5.0,   median   6.0,   max   6.0
+EntryRowCustomTraceMs                                min  10.0,   median  11.7,   max  16.1
```

Comment:
Recommend an empty line between startup and frame timing benchmarks. They're hard to distinguish as they are already.